### ISSUE TITLE
Not working on Windows 10 10.0.19043

### ISSUE THREAD

**Issue by n0rb4k** [klinix5/ProfSvcLPE#2]
**System Details:**
OS Name:   Microsoft Windows 10 Pro for Workstations
OS Version: 10.0.19043 N/A Build 19043
Hotfix(s):  KB5005539
                KB5000736
                KB5005716
                KB5001330
                KB5005699

**Problem:**                              
When the PoC is executed, it remains infinitely looping, searching for files in "C:\_tmp_" but it found nothing and remains in this state. The following is a screenshot to get it more clear:
![image](https://user-images.githubusercontent.com/22282280/139526207-dfc95855-99b5-4194-8023-27660cdf8910.png)

Any idea of what could be the cause for this unwanted behavior?

Thank you so much.

**Comment by klinix5** [comment]
Tell me again, why you're naming your user "TEMP" ?

**Comment by n0rb4k** [comment]
Of course. I tried many different approaches to get the PoC working. I mean, it has not been always executed with a user named "TEMP" but other names such as "user1", "user2", etc...

However, using the ProcMon I saw events like "C:\Users\Temp\AppData\" - NOT FOUND and I thought I would need the User's folder structure to successfully execute the PoC.

Honestly, I don't think this would be the problem, since I have tried other approaches as well.

Thank you so much for your attention.
